distribution/lib/Standard/Base/0.0.0-dev/src/Random.enso

```diff
@@ -236,6 +246,11 @@ type Random
 
     ## PRIVATE
        Return the default instance.
+
+       This method should never be public, since it has to create a
+       `Random_Generator` each time, and so it would be possible to use this to
+       create multiple such default instances at the same time, in which case
+       setting the seed for one would not affect the others.
```

Comment:
I think we could solve this in a way that does not require 'users' of the library (which I don't mean our end-users but just us using the Random module across the stdlib) to have to be so careful.

Reply:
Done